index.html

```diff
@@ -0,0 +1,291 @@
+              <h2>Grain+Glory</h2>
+            </div>
+            <ul class="btn">
+              <li class="ruby">Ruby on rails</li>
+              <li class="css">css</li>
+              <li class="js">Javascript</li>
+              <li class="html">html</li>
+            </ul>
+            <button type="button" class="btn-project">See Project</button>
+          </div>
+        </li>
+        <li class="card">
+          <div class="img_container"></div>
+          <div class="bottom">
+            <div class="intro">
+              <h2>Mult-Post Stories</h2>
+              <h2>Grain+Glory</h2>
+            </div>
+            <ul class="btn">
+              <li class="ruby">Ruby on rails</li>
+              <li class="css">css</li>
+              <li class="js">Javascript</li>
+              <li class="html">html</li>
+            </ul>
+            <button type="button" class="btn-project">See Project</button>
+          </div>
+        </li>
+      </ul>
+    </section>
+    <section id="about">
+      <div class="text">
+        <div class="text-img">
+          <h2 class="about-head">About me</h2>
+          <img
+            id="illus-img"
+            src="./images/illustration.svg"
+            alt="illustration"
+          />
+        </div>
+        <p id="description">
+          I can help you build a product , feature or website Look through some
+          of my work and expreince! If you like what you see and have a project
+          you need coded, don't hestitate to contact me.
+        </p>
+      </div>
+
+      <div class="resume">
+        <button type="button" class="btn-resume">Get My Resume</button>
+      </div>
+      <ul class="about-cards">
+        <li class="about-card">
+          <img class="icon_language" src="./images/icon-languages.svg" alt="" />
+          <h2 class="language">Languages</h2>
+          <ul class="languages-list">
+            <li id="jvscr">Javascript</li>
+            <li id="rub">Ruby</li>
+            <li id="htm">Html</li>
+            <li id="cs">Css</li>
+          </ul>
+        </li>
+        <li class="about-card">
+          <img
+            class="icon_language"
+            src="./images/icon-frameworks.svg"
+            alt=""
+          />
+          <h2 class="language">Frameworks</h2>
+          <ul class="languages-list">
+            <li id="bstrap">Bootstrap</li>
+            <li id="rub-rails">Ruby on rails</li>
+            <li id="spec">Rspec</li>
+            <li id="capy">Capybara</li>
+            <li id="sel">Selenium</li>
+          </ul>
+        </li>
+        <li class="about-card">
+          <img class="icon_language" src="./images/icon-skills.svg" alt="" />
+          <h2 class="language">Skills</h2>
+          <ul class="languages-list">
+            <li id="codekit">CodeKit</li>
+            <li id="Github">Github</li>
+            <li id="codepn">Codepen</li>
+            <li id="gitlab">Gitlab</li>
+            <li id="terminal">Terminal</li>
```

Comment:
- [ ] Always use the class attribute for multiple elements (do not use ID selector for multiple elements). As I have mentioned above the id attribute is a unique identifier that is used to specify an element in the document. It is used by CSS and JavaScript to perform a certain task for a unique element. You can read more about [here](https://css-tricks.com/the-difference-between-id-and-class/)

- [ ] Notice that your `li` elements share most of their styles, so a good idea would be to have a single class for all of them where you define common properties and then add another class to each if necessary to apply specific styles.